Samples/TrendingMovies/TrendingMovies/Utilities/Tracer.swift

```diff
@@ -30,8 +30,11 @@ extension Tracer {
         SentrySDK.start { options in
             options.dsn = "https://a92d50327ac74b8b9aa4ea80eccfb267@o447951.ingest.sentry.io/5428557"
             options.environment = "integration-tests"
-            options.enableProfiling = true
             options.debug = true
+
+            if ProcessInfo().arguments.contains("--io.sentry.enable-profiling") {
+                options.enableProfiling = true
+            }
```

Comment:
`m`: Don't we want to have this enabled by default for playing around with the sample app? We could invert this flag and pass an argument when we want disable it.

Reply:
Good call.